heartbeat/monitors/active/dialchain/builder.go

```diff
@@ -37,6 +37,7 @@ type Builder struct {
 	template         *DialerChain
 	addrIndex        int
 	resolveViaSocks5 bool
+	meta             common.MapStr
```

Comment:
Is this used anywhere? I'd prefer to not use common.MapStr if it's not really required.

Reply:
Good catch, removing